x-pack/plugins/apm/public/components/app/service_details/service_detail_tabs.tsx

```diff
@@ -23,7 +23,7 @@ import { ServiceMetrics } from '../service_metrics';
 import { ServiceNodeOverview } from '../ServiceNodeOverview';
 import { ServiceOverview } from '../service_overview';
 import { TransactionOverview } from '../transaction_overview';
-import { useApmService } from '../../../hooks/use_apm_service';
+import { useApmService } from '../../../context/apm_service/use_apm_service';
```

Comment:
In my mind this folder structure makes it clear that `useApmService` is a hook that consumes a context. It wasn't possible to see before.